spec/rubocop/cop/lint/void_spec.rb

```diff
@@ -225,6 +253,34 @@
     expect_no_corrections
   end
 
+  it 'registers an offense for void constant `CONST` within conditional if not on last line' do
+    expect_offense(<<~RUBY)
+      CONST = 5
+      unless condition
+        CONST
+        ^^^^^ Constant `CONST` used in void context.
+      end
+      top
+    RUBY
+
+    # `CONST unless condition` might indicate missing `return` or assignment,
```

Comment:
Looking at L210, should it be a comment like the following?

```suggestion
    # `unless condition; CONST end` might indicate missing `return` or assignment,
```